src/CodeGen_Xtensa.cpp

```diff
@@ -4057,6 +4105,10 @@ void CodeGen_Xtensa::visit(const Allocate *op) {
             }
         }
 
+        if (op->type.bits() == 24) {
```

Comment:
This doesn't look right, looks like workaround for something and should be done in schedule instead.